catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -3125,6 +3130,15 @@ private String buildIngestLog(CreateRequest createReq) {
         }
     }
 
+    private String getAttributeStringValue(Metacard mcard, String attribute) {
+        Attribute attr = mcard.getAttribute(attribute);
+        if (attr != null && attr.getValue() != null) {
+            return attr.getValue()
```

Comment:
Could you enforce that elsewhere? Throw an exception if a non-string attribute is used?